pkg/engine/renderer/parser_test.go

```diff
@@ -52,7 +52,7 @@ spec:
         - containerPort: 80
         env:
         - name: PARAM_ENV
-          value: "1"`)
+          value: 1`)
```

Comment:
```suggestion
          value: "1"`)
```
Please leave the quotes in, the value [needs to be a string](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#envvar-v1-core) for this to be parsed in as a `Deployment`. The reason it's not causing the test to fail is because we have the [fallback to `Unstructured`](https://github.com/kudobuilder/kudo/blob/615aeb88212b12471ca8394908e6d925bcb9b3d7/pkg/engine/renderer/parser.go#L31-L37).

Reply:
Fixes, thanks for the explanation.

Reply:
OTOH, shouldn't the test fail in that case? It didn't.

Reply:
Note my last sentence in the comment above.

Reply:
But the test checks that the kind of the resulting object is `Deployment`.